vendor.mod

```diff
@@ -177,7 +177,7 @@ replace (
 	github.com/matttproud/golang_protobuf_extensions => github.com/matttproud/golang_protobuf_extensions v1.0.1
 	github.com/prometheus/client_golang => github.com/prometheus/client_golang v1.6.0
 	github.com/prometheus/procfs => github.com/prometheus/procfs v0.0.11
-	github.com/vishvananda/netlink => github.com/vishvananda/netlink v1.1.0
+	github.com/vishvananda/netlink => github.com/vishvananda/netlink v1.2.1-beta.2
```

Comment:
oh! I should've looked at the diff; could you remove this `replace` (per the earlier discussion?)

Reply:
Yeah sure. I could have sworn I did that.